localization/camera/CMakeLists.txt

```diff
@@ -46,16 +53,26 @@ create_tool_targets(DIR tools
   INC ${INCLUDES}
 )
 
-create_test_targets(DIR test
-  LIBS camera ${CERES_LIBRARIES}
-  INC ${INCLUDES} ${CERES_INCLUDE_DIRS}
-)
+if(CATKIN_ENABLE_TESTING)
+  find_package(rostest REQUIRED)
 
-catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES camera
-  CATKIN_DEPENDS
-#  DEPENDS system_lib
-)
+  add_rostest_gtest(test_camera_model
+    test/test_camera_model.test
+    test/test_camera_model.cc
+  )
+  target_link_libraries(test_camera_model
+    camera
+  )
+
+  find_package(rostest REQUIRED)
```

Comment:
Why are there multiple find_package rostests? Our standard has been to import most things in the top level CMakeLists.txt, could we move this there?

Reply:
Thanks for noticing the repeated includes! Definitely a copy paste error.
I don't think we have the package rostest when cross-compiling, that's why it is inside the CATKIN_ENABLE_TESTING flag. This is a convention across all unit tests in the repo.